README.md

```diff
@@ -82,7 +82,7 @@ collect.slave_status                                   | 5.1           | Collect
 collect.heartbeat                                      | 5.1           | Collect from [heartbeat](#heartbeat).
 collect.heartbeat.database                             | 5.1           | Database from where to collect heartbeat data. (default: heartbeat)
 collect.heartbeat.table                                | 5.1           | Table from where to collect heartbeat data. (default: heartbeat)
-
+collect.all                                            | -             | Collect all metrics.
```

Comment:
I don't see the point of this flag.  We collect all enabled collectors by default.

Reply:
Maybe wording is bad. This enables all collectors. By default only 4 is enabled. To enable all other you need to pass them one by one. It's fine, but for testing it's easier to just turn one flag. https://github.com/prometheus/mysqld_exporter/blob/master/mysqld_exporter.go#L37-L41

Reply:
I don't think this is a feature we want to support.